internal/pkg/cli/interfaces.go

```diff
@@ -482,3 +483,12 @@ type svcInitializer interface {
 type roleDeleter interface {
 	DeleteRole(string) error
 }
+
+type clusterGetter interface {
+	Cluster(app, env string) (string, error)
+}
+
+type ecsClient interface {
```

Comment:
Can this name be more descriptive? 
```suggestion
type ecsTaskManager interface {
```